deploy/standard/infra/modules/vnet-peering.bicep

```
@@ -10,7 +10,7 @@ resource main 'Microsoft.Network/virtualNetworks@2024-01-01' existing = {
 }
 
 resource destinationToSourcePeering 'Microsoft.Network/virtualNetworks/virtualNetworkPeerings@2024-01-01' = {
-  name: 'hub-to-vnet'
+  name: vnetName

```

This should probably be something like 
```suggestion
  name: 'hub-to-${vnetName}`

```

---

Will address in a later PR.